.github/workflows/test.yml

```diff
@@ -1,12 +1,19 @@
 name: CI
 
-on: [push]
+on: [push, pull_request_target]
 
 jobs:
   build:
     runs-on: ubuntu-18.04
     container:
       image: horizonrobotics/alf:0.0.3-pytorch
+    # always run on push events, but only run on pull_request_target events
+    # when the pull requests are from fork repositories; for pull requests
+    # within the same repository, the push event is sufficient
+    if: >
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request_target' &&
+      github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
use parenthesis to group operations to make it less confusing

Reply:
done